src/components/MenuItem/MenuItem.module.css

```diff
@@ -35,6 +41,11 @@ limitations under the License.
   grid-template: "icon ." auto / auto 1fr;
 }
 
+/* Items with interactive controls like toggles are larger by default */
+label.item {
+  min-inline-size: 246px;
```

Comment:
I'm never a fan of those magic numbers. @americanrefugee or @janogarcia how did we land on that value for the menu width? Trying to come up with a formula that would convey a bit more context.

@robintown let's not block this PR on this detail though

Reply:
Removed it